packages/core-sdk/src/abi/sdkEntities.json

```diff
@@ -1,34 +1,39 @@
 [
   "ipAsset",
   "execute",
-
```

Comment:
I organized this file by different resource types and use one line to separate them. Also delete a few unused ones. All events are put in the last.  
Can you just add the functions you need for the dispute? Otherwise I have to redo and make it work again